Nodejs-Open-Collective.md

```diff
@@ -0,0 +1,36 @@
+# Node.js Open Collective
+
+The project maintains an Open Collective that is used to accept donations.
+
+The funds are to be used for specific efforts as decided by the TSC and
+documented on this page.
+
+## Account details
+
+The account is manged by the TSC. The user id and password
+are shared with TSC members through 1password.
+
+## Disbursements - management
+
+Potential disbursements will be agreed through TSC discussion
+and before approval, documented in an PR in the
+[TSC repository](https://github.com/nodejs/TSC) which adds
+the intended use and funding envelope to this page. The disbursement
+is considered approved if there's no objections from a project member
+in 72 hours, and at least one approval from a TSC voting member.
+Objections can be overridden with a TSC vote.
```

Comment:
```suggestion
is considered approved if there's no objections from a project member
in 72 hours, and consensus has been reached in the TSC. Objections can be
overridden with a TSC vote.
```

Reply:
@aduh95 I merged your suggetion, but have added one as I think we need TSC consensus before funds are allocated. In my opinion having one TSC approval is not enough. We need TSC consensus which is a higher bar and will take a few weeks which I think is important in this case.

Reply:
SGTM